cli/fathom_web/accuracy.py

```diff
@@ -28,76 +28,104 @@ def accuracy_per_tag(y, y_pred):
         return (successes / number_of_tags), false_positives, false_negatives
 
 
+# Max chars from a start tag we're willing to display in the per-tag metrics
+# report:
+MARKUP_MAX_WIDTH = 36
```

Comment:
Since you're returning a slice of max size 36 in the [FathomFox PR](https://github.com/mozilla/fathom-fox/pull/75/files#diff-ce5ff8cbb3b5f2aa37a784e7132e2a95R122l), do you need this constant here?

Reply:
It's belt and suspenders. As I said in 1cccb87fe3d90531cdda81586ef560d1b6a72e8d's commit message, "the exact length may change someday, and I don't want the layout to break".